### ISSUE TITLE
[feature request] Step function keys work when pointing at the code

### ISSUE THREAD

**Issue by nww02** [Dotneteer/spectnetide#94]
When I'm debugging, the focus often shifts back from the ZX VM to the code window, especially when a breakpoint is hit.  If I'm using F10 to step, or F5 to skip through breakpoints, then the F keys stop stepping when focus moves away from the VM.  To use them, I have to click on the VM to focus on it first.

This kinda makes them less useful, and I find myself clicking with the mouse to step / run.

It would be great if the F5 (see feature request 93), F11 and F10 keys also worked when editing (or just focusing on) the code.

**Comment by Dotneteer** [comment]
Several months ago I had a run around this feature. I could solve using the F keys while you are in the Z80 Disassembly or ZX Spectrum Memory window, but I could not take the control when you pressed F keys while in the code window. The issue is that VS IDE code window catches the F keys. Nonetheless, I try to find some fresh solutions...

**Comment by nww02** [comment]
Thanks!
  It would really help the usability and ease of debugging, as at the moment it involves a lot of clicking to overcome! Thank you for considering this! :)

**Comment by Dotneteer** [comment]
Well, after reading a lot of documentation and forums, I recognized that implementing a feature that can catch F5, F10, F11 (and any other function keys normally assigned by VS IDE to a project system specific command, such as Start, Start without Debugging) is a real nightmare. I had to implement a number of classes with about hundred lines of code -- and its an undiscovered area for me.

Nonetheless, I find a tricky workaround :-)

Right now, whey the code reaches a breakpoint, I display the code file and move the focus to the code window. Actually, this is what prevents you using the F keys. My workaround would be this:
* After displaying the code with the current position, I turn the focus back to the ZX Spectrum Emulator window.
* Right now, the ZX Spectrum Emulator window accepts the F keys, just as the Z80 Disassembly window. I extend this behavior to the ZX Spectrum Memory and Z80 Registers window (and probably to other tool windows later). So if you are in any of these windows, the F keys will work.

There's a little price for this workaround: Whenever you work in the code window (you move the focus there), the F keys won't work unless you turn the focus back to one of the previously mentioned tool windows.
